docs/source/gbd2017_models/risk_attributable_causes/diabetes_mellitus/index.rst

```diff
@@ -189,19 +197,19 @@ State and Transition Data Tables
      - Notes
    * - 1
      - S
-     - S
+     - C
      - incidence_c587
      -
    * - 2
      - C
      - M
-     - :math:`\sum_{s\in \text{incidence_sequelae_mod.sub_causes.c587}}`
-     - = incidence of Diabetes Mellitus Type 1 uncomplicated sequelae + incidence of Diabetes Mellitus Type 2 uncomplicated sequelae
+     - :math:`\frac{\sum_{s\in \text{prevalence_sequelae_mod.sub_causes.c587}} \times\ \text{incidence_c587}}{\text{prevalence_c587}}`
```

Comment:
Do you intend C to be a "transient state"?  In other words, will there ever be simulants who are in state C at the beginning of a timestep, or upon entering C will each simulant immediately progress to M or Sev?

We need a way to note this in the transition table, and then the transition probability will not need to include the incidence_c587 term.

Reply:
yes Abie - "state" C is not really a state, it is transient.

Reply:
@davidbernotas , if we are not including incidence or transition between states then how will a simulant go from 'S'>'C'>'M' or 'Sev'?